plugins/org.jkiss.dbeaver.ext.sqlmx/src/org/jkiss/dbeaver/ext/sqlmx/model/SqlmxDataSource.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.sqlmx.model;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.generic.model.GenericDataSource;
+import org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel;
+import org.jkiss.dbeaver.model.DBPDataSourceContainer;
+import org.jkiss.dbeaver.model.connection.DBPConnectionConfiguration;
+import org.jkiss.dbeaver.model.exec.DBCException;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+import java.sql.Connection;
+import org.jkiss.dbeaver.Log;
+
+/**
+ * Sqlmx datasource
+ */
+public class SqlmxDataSource extends GenericDataSource {
+    private static final Log log = Log.getLog(SqlmxDataSource.class);
+    public SqlmxDataSource(DBRProgressMonitor monitor, DBPDataSourceContainer container, GenericMetaModel metaModel)
+        throws DBException {
+        super(monitor, container, metaModel, new SqlmxSQLDialect());
+//FJ        log.debug ("****** SqlmxDataSource called ");
+    }
+
+    @Override
+    protected boolean isPopulateClientAppName() {
+    	// SQL/MX drivr does not support this type of settingthe application name
+        return false;
+    }
```

Comment:
It's automatically set to `false` for all data sources that inherit from `GenericDataSource`

https://github.com/dbeaver/dbeaver/blob/268867bb73665954d082ca2e6191b44c8ad5dac3/plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/GenericDataSource.java#L211-L213